docs/en/observability/monitor-kubernetes.asciidoc

```diff
@@ -0,0 +1,223 @@
+
+TODO: Provide a brief intro. We'll need to decide how much to cover here vs
+the monitoring overview. Again lots of good introductory info here:
+https://www.elastic.co/blog/kubernetes-observability-tutorial-k8s-log-monitoring-and-analysis-elastic-stack)
+
+[discrete]
+=== Logs you should monitor
+
+TODO: Discuss the various logs (Kubernetes cluster, nodes, host, container logs)
+that users might want to collect. I'm not sure if this deserves a separate
+section or should be part of the previous section.
+
+[discrete]
+=== Deploy {filebeat} to collect logs
+
+TODO: Describe how to configure and deploy Filebeat as a daemonset. We should
+walk users through the parts of the deployment YAMLs, explain what they do, and point
+out the settings they might want to change or add. Rough list of what
+we need to cover:
+
+* Download the Filebeat deployment manifest
+
+* Set the connection information for Elasticsearch (also describe how to create
+secrets)
+
+* Configure log collection:
+
+** Collect Kubernetes logs (cluster, node)
+
+** Collect host logs
+
+** Collect container logs (use autodiscovery)
+We should probably explain what the default config with the container input does
+then focus on documenting how to use autodiscovery. 
+
+* Add metadata to events. Enrich the event with metadata coming from Docker,
+Kubernetes, host, and the cloud providers.
+
+* (optional) Drop unwanted events
+
+* Deploy Filebeat as a DaemonSet on Kubernetes
+
+[discrete]
+=== View logs in {kib}
+
+TODO: Describe how to view logs in Kibana. Show how to use the Logs app and how to
+view pre-built dashboards and visualizations.
+
+[discrete]
+== Part 2: Monitor health and performance metrics
+
+TODO: provide a brief intro. We'll need to figure out how much to cover here vs
+the monitoring overview. Again lots of good introductory info under
+"K8s metrics collection with Metricbeat"
+https://www.elastic.co/blog/kubernetes-observability-tutorial-k8s-metrics-collection-and-analysis[here].
+
+
+[discrete]
+=== Metrics you should monitor
+
+TODO: Discuss the metrics that users should monitor: cluster-level
+metrics from Kubernetes, pod metrics, metrics about nodes, technology-specific
+metrics (like nginx),etc. Explain where the metrics come from (kubelet API,kube-state-metrics,
+apiserver, etc) and why to monitor them.
+
+
+[discrete]
+=== Deploy {metricbeat} to collect metrics
+
+TODO: Describe how to configure and deploy Metricbeat as a daemonset. Rough list
+of what we need to cover:
+
+* Download the Metricbeat deployment manifest
+
+* Set the connection information for Elasticsearch (also describe how to create
+secrets or point to earlier section for reminder)
+
+* Configure metrics collection:
+
+** Collect system metrics.
+
+** Collect Docker metrics.
+
+** Collect Kubernetes pod metrics.
+
+** Collect Kubernetes state metrics
+
+** Collect application-specific metrics (use hint-based autodiscovery).
+Examples: NGINX, MySQL
+
+** Collect Prometheus metrics.
```

Comment:
What exactly do we mean with this? What kind of Prometheus metrics? For instance what `kube-state-metrics` serves is metrics in Prometheus format, so to what Prometheus metrics we need to refer to explicitly?

Reply:
Sorry, the heading is misleading. :-( It should say "Collect Metrics from Prometheus" (basically what you get when you configure the Prometheus module).

Reply:
I understand it's about collecting Prometheus metrics from applications running in the cluster. I think it makes sense to have a section for that in this context, perhaps rewording to make it more clear